service/src/tests/mock.rs

```diff
@@ -3,8 +3,8 @@ use itp_types::{Enclave, ShardIdentifier, H256 as Hash};
 
 pub struct TestNodeApi;
 
-pub const W1_URL: &str = "127.0.0.1:2222";
-pub const W2_URL: &str = "127.0.0.1:3333";
+pub const W1_URL: &str = "127.0.0.1:22222";
+pub const W2_URL: &str = "127.0.0.1:33333";
```

Comment:
Unrelated to the features in this PR: I had to change the ports for the test websockets, they collided with my WSL SSH-daemon